src/QuiqChatClient.js

```diff
@@ -2,22 +2,25 @@
 import * as API from './apiCalls';
 import {setGlobals, checkRequiredSettings} from './globals';
 import {connectSocket, disconnectSocket} from './websockets';
-import type {AtmosphereMessage, Message, ApiError, UserEventTypes} from './types';
+import type {AtmosphereMessage, TextMessage, ApiError, UserEventTypes, Event} from './types';
 import {MessageTypes, quiqChatContinuationCookie} from './appConstants';
 import {set, get} from 'js-cookie';
-import {differenceBy, last} from 'lodash';
+import {differenceBy, last, partition} from 'lodash';
+import {sortByTimestamp} from './utils';
 
-const getConversationMessages = async () => {
+const getConversation = async (): Promise<{events: Array<Event>, messages: Array<TextMessage>}> => {
```

Comment:
I mean there's not much else in the conversation, it just might be a confusing flow error if we're not paying attention later. Not a big deal though, it can stay